### PR TITLE
docs: update widgets example

### PR DESCRIPTION
**Pull request by samouss**
**Summary**

This PR fix some issues related to the example in the widgets:

- consistent name across the examples
- correct indentation
- correct rendering (always display something)

### CODE REVIEW

File: packages/react-instantsearch/src/core/Index.js [hunk 1]
```diff
@@ -11,35 +11,37 @@ import React, { Component, Children } from 'react';
...
- * }
+ * const App = () => (
+ *   <InstantSearch
+ *     appId="latency"
+ *     apiKey="6be0576ff61c053d5f9a3225e2a90f76"
+ *     indexName="ikea"
+ *   >
+ *     <Configure hitsPerPage={5} />
+ *     <SearchBox />
+ *
```
Comment (Haroenv) [inline packages/react-instantsearch/src/core/Index.js]: here's the other spacing

File: packages/react-instantsearch/src/widgets/Breadcrumb.js [hunk 2]
```diff
@@ -59,26 +59,32 @@ import Breadcrumb from '../components/Breadcrumb';
...
+ *     indexName="ikea"
+ *   >
+ *     <Breadcrumb
+ *       attributes={[
+ *         'category',
+ *         'sub_category',
+ *         'sub_sub_category',
+ *       ]}
+ *     />
+ *
```
[the remark this answers is not in the document]
Comment (samouss) [reply packages/react-instantsearch/src/widgets/Breadcrumb.js]: It's for clarity actually. But I can remove them.

[the remark this answers is not in the document]
Comment (Haroenv) [reply packages/react-instantsearch/src/widgets/Breadcrumb.js]: I understand why you had them, just noting it was a bit inconsistent with only two examples having them

**Comment by vvo** [comment]
GG nice chore work!

**Pull request merged**